docs/plugins/getting-started.md

```diff
@@ -89,6 +89,36 @@ ID (the one you set in `manifest.json`).
 
 :::
 
+## Development Companion Extension
+
+Replugged v4.3.0 contains support for
+[Vencord's dev companion extension](https://marketplace.visualstudio.com/items?itemName=Vendicated.vencord-companion).
+It is an extension for VSCode that allows you to test your module finds and plaintext patches
+without having to rebuild your plugin and reload your client.
+
+You can find the extension in VSCode's extension store when looking for "Vencord", and after
```

Comment:
I think this part can be removed, since now there is the direct link to the extension.